x-pack/plugins/enterprise_search/public/applications/analytics/components/add_analytics_collections/add_analytics_collection_form.tsx

```diff
@@ -9,103 +9,46 @@ import React from 'react';
 
   return (
-    <EuiFlexGroup>
-      <EuiFlexItem>
-        <EuiPanel hasShadow={false} paddingSize="xl" grow={false} hasBorder>
-          <EuiTitle>
-            <h2>
-              {i18n.translate('xpack.enterpriseSearch.analytics.collectionsCreate.form.title', {
-                defaultMessage: 'Create an analytics collection',
-              })}
-            </h2>
-          </EuiTitle>
-          <EuiSpacer size="s" />
-          <EuiText size="s">
-            <p>
-              {i18n.translate('xpack.enterpriseSearch.analytics.collectionsCreate.form.subtitle', {
-                defaultMessage:
-                  'An analytics collection provides a place to store the analytics events for any given search application you are building. Give it a memorable name below.',
-              })}
-            </p>
-          </EuiText>
-          <EuiSpacer />
-          <EuiForm
-            component="form"
-            onSubmit={(e) => {
-              e.preventDefault();
-              if (canSubmit) {
-                createAnalyticsCollection();
-              }
-            }}
-          >
-            <EuiFormRow label="Name">
-              <EuiFieldText
-                name="collection-name"
-                fullWidth
-                autoFocus
-                value={name}
-                onChange={(e) => {
-                  setNameValue(e.target.value);
-                }}
-              />
-            </EuiFormRow>
-            <EuiSpacer />
-            <EuiFlexGroup justifyContent="flexEnd" gutterSize="s">
-              <EuiFlexItem grow={false}>
-                <EuiButtonEmpty
-                  onClick={() => {
-                    navigateToUrl(ROOT_PATH);
-                  }}
-                >
-                  {i18n.translate(
-                    'xpack.enterpriseSearch.analytics.collectionsCreate.form.cancelButton',
-                    {
-                      defaultMessage: 'Cancel',
-                    }
-                  )}
-                </EuiButtonEmpty>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiButton fill type="submit" isLoading={isLoading} isDisabled={!canSubmit}>
-                  {i18n.translate(
-                    'xpack.enterpriseSearch.analytics.collectionsCreate.form.continueButton',
-                    {
-                      defaultMessage: 'Continue',
-                    }
-                  )}
-                </EuiButton>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiForm>
-        </EuiPanel>
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <EuiForm
+      id={formId}
+      component="form"
+      onSubmit={(e) => {
+        e.preventDefault();
+        if (canSubmit) {
+          createAnalyticsCollection();
+        }
+      }}
+    >
+      <EuiFormRow label="Collection name" isInvalid={!!inputError} error={inputError}>
```

Comment:
label missing translation

Reply:
This translation is still missing. Can you make sure to add this in a subsequent PR @yansavitski?